ignition/service/infrastructure.py

```diff
@@ -85,6 +114,12 @@ def find_infrastructure(self, template, template_type, instance_name, deployment
         :param str instance_name: name given as search criteria
         :param dict deployment_location: the deployment location to deploy to
         :return: an ignition.model.infrastructure.FindInfrastructureResponse
+
+        :raises:
+            ignition.service.infrastructure.InvalidInfrastructureTemplateError: if the Template is not valid
+            ignition.service.infrastructure.UnreachableDeploymentLocationError: the Deployment Location cannot be reached
+            ignition.service.infrastructure.TemporaryInfrastructureError: there is an issue handling this request at this time, an attempt should be made again at a later time
```

Comment:
"an attempt should be made again at a later time" implies that it's the responsibility of the client to do this, maybe say that Ignition will re-attempt the request at a later time?